### PR TITLE
tweak punctuation, wording, etc.

### PR DESCRIPTION
**Pull request by RichMorin**
The change includes numerous small tweaks to punctuation, wording, etc.  I didn't try to change "...and include directories generated build files." because I can't figure out what it's supposed to mean.  Help?

### CODE REVIEW

**Comment by merge** [comment]
> The change includes numerous small tweaks to punctuation, wording, etc. I didn't try to change "...and include directories generated build files." because I can't figure out what it's supposed to mean. Help?

@skirk you added this sentense during https://github.com/libts/tslib/pull/137/files what did you mean?

**Comment by merge** [comment]
thanks a lot for looking over it! I'm not convined about all the oxford commas, but I'm leaning towards merging...

**Comment by skirk** [comment]
> > The change includes numerous small tweaks to punctuation, wording, etc. I didn't try to change "...and include directories generated build files." because I can't figure out what it's supposed to mean. Help?
> 
> @skirk you added this sentense during https://github.com/libts/tslib/pull/137/files what did you mean?

I suppose it should read ".. to the generated build files". To be fair, I think the whole sentence can be omitted or replaced with something like:
The library adds `tslib::tslib` target and all the client needs to do is to link against it.

File: README.md [hunk 1]
```diff
@@ -723,25 +724,25 @@ calibration (linear filter) and the infinite impulse response filter in ts.conf.
 
 ### CMake
 
-Alternatively you can use CMake to build the project.
-To create an build in the project tree:
+Alternatively, you can use CMake to build the project.
+To create building in the project tree:
```
Comment (skirk) [inline README.md:728]: This ought to be "a build"

**Comment by RichMorin** [comment]
> On Sep 15, 2020, at 05:47, Martin Kepplinger <notifications@github.com> wrote:
> 
> thanks a lot for looking over it! I'm not convined about all the oxford commas, but I'm leaning towards merging...

"I'd like to thank my parents, God and Mother Teresa."

-r

File: README.md (hunk 1 above)
Comment (merge) [reply README.md:728]: If you want to fix that and the above replaced sentence on top now, go ahead. Otherwise I might do it soon. thanks!

**Pull request merged**